client/src/contexts/HomePageContext.tsx

```diff
@@ -0,0 +1,52 @@
+import { createContext, ReactChild, ReactChildren, useState } from 'react'
+import {
+  DEFAULT_QUESTIONS_DISPLAY_STATE,
+  DEFAULT_QUESTIONS_SORT_ORDER,
+  QuestionsDisplayState,
+  QuestionsSortOrder,
+} from '../components/Questions/questions'
+
+interface HomePageProps {
+  children: ReactChild | ReactChild[] | ReactChildren | ReactChildren[]
```

Comment:
Was wondering if this can be replaced with JSX.Element?

Reply:
oh yeah I replaced with `ReactNode`